android/app/src/main/NetworkSecurityConfig.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Ensure that the NetworkSecurityConfig.xml file is located in the correct directory: your_project_directory/android/app/src/main/res/xml/.